interface/new/new.php

```diff
@@ -1,242 +1,3 @@
-  </td>
-  <td>
-   <input type='entry' size='15' name='fname' value='<?php echo $form_fname; ?>'>
-  </td>
- </tr>
-
- <tr>
-  <td>
-   <span class='bold'><?php xl('Middle Name','e');?>: </span>
-  </td>
-  <td>
-   <input type='entry' size='15' name='mname' value='<?php echo $form_mname; ?>'>
-  </td>
- </tr>
-
- <tr>
-  <td>
-   <span class='bold'><?php xl('Last Name','e');?>: </span>
-  </td>
-  <td>
-   <input type='entry' size='15' name='lname' value='<?php echo $form_lname; ?>'>
-  </td>
- </tr>
-
- <tr>
-  <td>
-   <span class='bold'><?php xl('Sex','e'); ?>: </span>
-  </td>
-  <td>
-   <select name='sex'>
-    <option value=''>Unassigned</option>
-<?php
-$ores = sqlStatement("SELECT option_id, title FROM list_options " .
-  "WHERE list_id = 'sex' ORDER BY seq");
-while ($orow = sqlFetchArray($ores)) {
-  echo "    <option value='" . $orow['option_id'] . "'";
-  if ($orow['option_id'] == $form_sex) echo " selected";
-  echo ">" . $orow['title'] . "</option>\n";
-}
-?>
-   </select>
-  </td>
- </tr>
-
-<?php if ($GLOBALS['inhouse_pharmacy']) { ?>
- <tr>
-  <td>
-   <span class='bold'><?php xl('Referral Source','e'); ?>: </span>
-  </td>
-  <td>
-   <select name='refsource'>
-    <option value=''>Unassigned</option>
-<?php
-$ores = sqlStatement("SELECT option_id, title FROM list_options " .
-  "WHERE list_id = 'refsource' ORDER BY seq");
-while ($orow = sqlFetchArray($ores)) {
-  echo "    <option value='" . $orow['option_id'] . "'";
-  if ($orow['option_id'] == $form_refsource) echo " selected";
-  echo ">" . $orow['title'] . "</option>\n";
-}
-?>
-   </select>
-  </td>
- </tr>
-<?php } ?>
-
- <tr>
-  <td>
-   <span class='bold'><?php xl('Birth Date','e');?>: </span>
-  </td>
-  <td>
-   <input type='text' size='10' name='DOB' id='DOB' value='<?php echo $form_dob; ?>'/>
-  </td>
- </tr>
-
- <tr<?php echo $regstyle ?>>
-  <td>
-   <span class='bold'><?php xl('Registration Date','e');?>: </span>
-  </td>
-  <td>
-   <input type='text' size='10' name='regdate' id='regdate' value='<?php echo $form_regdate; ?>'/>
-  </td>
- </tr>
-
- <tr>
-  <td>
-   <span class='bold'><?php xl('Patient Number','e');?>: </span>
-  </td>
-  <td>
-   <input type='entry' size='5' name='pubpid' value='<?php echo $form_pubpid; ?>'>
-   <span class='text'><?php xl('omit to autoassign','e');?> &nbsp; &nbsp; </span>
-  </td>
- </tr>
-
- <tr>
-  <td colspan='2'>
-   &nbsp;<br>
-   <input type='submit' name='form_create' value=<?php xl('Create New Patient','e'); ?> />
-  </td>
-  <td>
-  </td>
- </tr>
-
-</table>
-</center>
-</form>
-<script language="Javascript">
-<?php
-if ($form_pubpid) { 
-  echo "alert('" . xl('This patient ID is already in use!') . "');\n";
-}
-?>
-</script>
-<script>
-    $(function() {
-        $("#DOB, #regdate").datetimepicker({
-            timepicker: false,
-            format: "<?= $DateFormat; ?>"
-        });
-        $.datetimepicker.setLocale('<?= $DateLocale;?>');
-    });
-</script>
-
-</body>
-</html>
+//Useless
```

Comment:
You preserved this file by reducing it to two lines?  Is this related to this PR at all?
-I will go back and read your description.  It looks like you may need to rebase your fork, bringing it up to sync with the head, then create a new branch off of your nice, clean, rebased master for this.  I believe it has changes from your previous commits.
  I will test the changes to the menu hover.